src/dynamic-plugin-sdk.ts

```diff
@@ -3,11 +3,14 @@ import * as React from 'react';
 import * as _ from 'lodash';
 import { HttpError } from './shared/utils/error/http-error';
 import { useDeepCompareMemoize } from './shared';
+import {
+  k8sListResourceItems,
+  k8sGetResource,
+} from '@openshift/dynamic-plugin-sdk-utils';
```

Comment:
Should be able to drop this file entirely. I was able to delete this variant in HAC-Core when I pulled alpha3

Reply:
We should identify those and if needed bring those over to the SDK... we shouldn't keep this file, it was very much a hack.

Reply:
They shouldn't need `commonFetch` either -- if they do need it they are not properly hooked into HAC-Core.